tests/test_analysis.py

```diff
@@ -0,0 +1,95 @@
+import covasim as cv

```

Comment:
Could you add a brief comment to the top of the file explaining what it's testing? Doesn't have to be long, see e.g. `test_interventions.py` for an example